scripts/perf-test/ec2-terminate.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
:pick: Add docs at the top for how to run this (e.g. it looks like it should be called `ec2-terminate.sh start|status`)